NEWS.md

```diff
@@ -1,60 +1,46 @@
 # xportr (development version)
 
-## New Features and Bug Fixes
-
-* `xportr_metadata()` can set `verbose` for a whole pipeline, i.e. setting `verbose` in `xportr_metadata()` will populate to all `xportr` functions.  (#151)
+## New Features
 
-* All `xportr` functions now have `verbose = NULL` as the default (#151)
-
-* Bug fix for domain filtering (#137)
-
-## Documentation
+* All core functions can be run together by using new function `xportr()` (#137)
 
```

Comment:
Can we remove the empty lines between sub-sections?

It seems odd.

Reply:
i have prettified

Reply:
Oh, I meant the empty lines between items (starting with `*`). The ones between headings and the first item seem fine.

![image](https://github.com/atorus-research/xportr/assets/211358/f70e94df-2d7e-42f4-b09e-bad311d1d5d5)

I only mentioned this as previous releases don't have it :shrug: This is very minor though, release is looking goooood!

Reply:
ah i see!!  updated!  I appreciate the detailed being paid attention to the changelog!!